srcpkgs/xplr/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'xplr'
+pkgname=xplr
+version=0.14.2
+revision=1
+archs="x86_64* i686* aarch64* armv[67]l* armv[67]hf*"
+hostmakedepends="tar"
+makedepends="libgcc-devel"
+short_desc="Hackable, minimal, fast TUI file explorer written in Rust"
+maintainer="Shubham Pawar <shubhampawar3007@gmail.com>"
+license="MIT"
+homepage="https://github.com/sayanarijit/xplr"
+distfiles="${homepage}/releases/download/v${version}/${pkgname}-linux.tar.gz
+	${homepage}/raw/v${version}/LICENSE
+	${homepage}/raw/v${version}/README.md
+	${homepage}/raw/v${version}/src/init.lua
+	${homepage}/raw/v${version}/assets/desktop/${pkgname}.desktop"
```

Comment:
Drop.
These files are all present in the source pkg and therefore also correct checksum to only one ..

Reply:
The [xplr](https://github.com/sayanarijit/xplr/releases/download/v0.14.2/xplr-linux.tar.gz) release includes only binary and noting else. I have used [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=xplr-bin) as reference

Reply:
and no source code? You can't get the source code somehow?

Reply:
PKGBUILDs, apparently, are not a good reference for Void packages. We prefer building from source. Please correct this defect in these templates.

Reply:
Bare with my ignorance, but the whole point of using void is to have binary packages and compile less as compared to arch, etc

Reply:
Binary packages are still built from some source. We prefer to be the ones to build from source, not just repackage files provided by upstream

Reply:
Then I don't think we need this template. Because we could just install xplr using cargo.